module/Frontpage/src/Frontpage/Service/Frontpage.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace Frontpage\Service;
+
+use Application\Service\AbstractAclService;
+
+/**
+ * Frontpage service.
+ */
+class Frontpage extends AbstractAclService
+{
+
+    /**
+     * Retrieves all data which is needed on the home page
+     */
+    public function getHomePageData()
+    {
+        $birthdayInfo = $this->getBirthdayInfo();
+        return array(
+            'birthdays' => $birthdayInfo['birthdays'],
+            'birthdayPhoto' => $birthdayInfo['photo']
+        );
+    }
+
+    /**
+     * Retrieves all birthdays happening today, which should be shown on the home page.
+     * Includes the age and a random photo of someone whom has a birthday.
```

Comment:
It shouldn't be a random photo. It should be the photo of the most active member (actually, the member with the most photos).